src/sentry/middleware/integrations/parsers/base.py

```diff
@@ -83,8 +84,11 @@ def get_response_from_control_silo(self) -> HttpResponseBase:
         return self.response_handler(self.request)
 
     def get_response_from_region_silo(self, region: Region) -> HttpResponseBase:
-        region_client = RegionSiloClient(region)
-        return region_client.proxy_request(incoming_request=self.request)
+        with metrics.timer(
+            "integration_proxy.control.get_response_from_region_silo", sample_rate=1.0
```

Comment:
We may also want to tag these with the destination region in the future too since control->US should be a lot faster than control->DE

I think `tags={"destination_region": region.name}` is all that'd be needed